website/templates/alert.mako

```diff
@@ -1,12 +1,17 @@
 <div id="alert-container">
-% for message, css_class, dismissible in status:
+% for message, css_class, dismissible, safe in status:
       <div class='alert alert-block alert-${css_class} fade in'>
         % if dismissible:
         <button type="button" class="close" data-dismiss="alert" aria-label="Close">
           <span aria-hidden="true">&times;</span>
         </button>
         % endif
-        <p>${message}</p>
+
+        % if safe:
```

Comment:
Discussed in person, but the semantics of the `safe` parameter is different from the `safe` parameter for the template renderer. Maybe rename to `trust` or `is_html`.